_data/reviewers_vol2_issue1.yml

```diff
@@ -1,4 +1,91 @@
-- name: Crista Lopes
-  affiliation: University of California, Irvine, California
-  extra: "Editor-in-Chief"
+- name: Guido Salvaneschi
+  affiliation: TU Darmstadt, Germany
+  extra: "Program Chair"
```

Comment:
Please change to "Associate Editor", which has been your role in context of the journal.
(see also
- https://github.com/programming-journal/programming-journal.github.io/blob/master/_data/reviewers_vol3_issue2.yml#L5)